doc/releases/luminous.rst

```diff
@@ -1,3 +1,274 @@
+v12.2.7 Luminous
+================
+
+This is the seventh bugfix release of Luminous v12.2.x long term
+stable release series. This release contains severalx fixes for
+regressions in the v12.2.6 and v12.2.5 releases.  We recommand that
+all users upgrade.
+
+:note: The v12.2.6 release has serious known regressions.  If you installed this release, please see the upgrade procedure below.
+
+:note: The v12.2.5 release has a potential data corruption issue with erasure coded pools.  If you ran v12.2.5 with erasure coding, please see below.
+
+Upgrading from v12.2.6
+----------------------
+
+v12.2.6 included an incomplete backport of an optimization for
+BlueStore OSDs that avoids maintaining both the per-object checksum
+and the internal BlueStore checksum.  A critical follow-on patch was
+not included, which introduced a bug that corrupted (failed to update)
+the stored per-object checksum value for some objects.  This can
+result in an EIO error when trying to read those objects.
+
+#. This problem only affects clusters with BlueStore.  If your cluster
+   uses FileStore only, no special action is required.
+
+#. If your cluster has only BlueStore OSDs (no FileStore), or your
+   cluster has a mix of BlueStore and FileStore and you are certain
+   that no OSDs have gone in or out and no PGs been rebalanced since
+   v12.2.6 was first installed, then you should enable the following
```

Comment:
I think we need osd_distrust_data_digest in any mixed cluster that ran v12.2.6.  Why do you think it matters if PG changed while running v12.2.6 and not when running v12.2.7 with osd_skip_data_data = true?

Here is a scenario.
1. Create/writefull object while running version earlier than 12.2.6 (12.2.5?) in mixed bluestore/filestore cluster
1. Install 12.2.6
1. Write data to object with a bluestore primary and filestore replicas ** you now have invalid data_digest on filestore
1. Upgrade to 12.2.7 and per instructions set osd_skip_data_digests = true but NOT osd_distrust_data_digest.
1. After the upgrade, primary of the pg you had written to goes down/out
1. Filestore replica becomes primary
1. Client tries to read whole file from replica and gets EIO (or tries to repair primary).

Reply:
In theory if the primaries are all teh same it would be safe, but that is probably to dangerous. Changed it!